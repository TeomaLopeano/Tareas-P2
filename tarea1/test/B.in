cantidadEnCadena
imprimirCadena
estaEnCadena 1
insertarAlInicio 9 9.0
cantidadEnCadena
imprimirCadena
estaEnCadena 1
estaEnCadena 9
insertarAlInicio 10 10.0
insertarAlInicio 7 7.0
insertarAlInicio 14 14.0
cantidadEnCadena
imprimirCadena
estaEnCadena 1
estaEnCadena 9
estaEnCadena 14
Fin