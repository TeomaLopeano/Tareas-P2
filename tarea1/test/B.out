1>La cantidad es 0.
2>
3>1 no está en la cadena.
4>Se insertó (9,9.00) al inicio de la cadena.
5>La cantidad es 1.
6>(9,9.00)
7>1 no está en la cadena.
8>9 está en la cadena.
9>Se insertó (10,10.00) al inicio de la cadena.
10>Se insertó (7,7.00) al inicio de la cadena.
11>Se insertó (14,14.00) al inicio de la cadena.
12>La cantidad es 4.
13>(14,14.00)(7,7.00)(10,10.00)(9,9.00)
14>1 no está en la cadena.
15>9 está en la cadena.
16>14 está en la cadena.
17>Fin.
